src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
```

Comment:
I think it is not just `Area`, you should find more appropriate name. [Hint](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#dont-use-verbs-for-classinterface-names).

Reply:
The issue with interface name is still actual, please refactor and you still need to find a place to use this interface